### ISSUE TITLE
VRO Team Sprint S

### ISSUE THREAD

**Issue by meganhicks** [department-of-veterans-affairs/abd-vro#2851]
# Sprint Sequoia National Park 

<img width="1279" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOEE5QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d6104e00cdb4ac4eed1731e98552fc64e51e09ae/image.png" alt="image.png" />

#### Facts: 
- Located in California 
- established on September 25, 1890
- established to protect giant sequoia trees, the largest living trees by volume on Earth.
- Megan's baby brother is there right now 

## Capacity: 
The percentages below reflect the amount of capacity the team members have to work towards sprint goals and priorities. Therefore, the sprint plan commitments should be reduced using these percentages. _**Note: includes time off, non-sprint commitments, and new team member ramp-up.**_

**All Team:** 46.9% 
**All Eng:** 42.9%
**All UX:** 71.4%

## Sprint Goal: 
At the end of this sprint, what do we want to have accomplished / delivered?
- Finish the deployment improvement work we did not get to in the Last Sprint and hold one another accountable to hitting sprint this sprint goal. 
- **Note from Berni:** These are our team's usual tasks and responsibilities, broken down by category: A) Bugs/Defects, B) Mature Partner Team Model, C) Keeping the lights on (VRO can further discuss how to define/determine what this means), D) Help Clients/Stakeholders reach their objectives

### Planned Issues (in Priority Order):

- [x] 1. Update on EE: Teja
- [x] 2. Update on CC : Erik 
- [x] 3. "Oncall" Secrel Work- Erik 
- [x] 4. Finish Rbac for Dev #2530 - Ponnia
- [ ] 5. Finish Rbac for QA #2847 - Ponnia and Cheng
- [x] 6. Tech Spec on Secrel workflow to allow signing on single images #2472
- [x] 7. Finish documenting VRO DB entities for partner team use of alembic for DB migrations #2329 - Teja 
- [ ] 8. Remove camel code in xample-workflow app #2745 - Cheng 
- [x] 9. Discovery for flickering integration tests #2824 - Cheng 
- [ ] 10. ~~Setup Datadog User for RDS Monitoring#2504- Ponnia (blocked and removed from Sprint S, move to future sprint)~~
- [ ] 11. Discovery on DORA #2825 - Open - Teja
- [ ] 12. Discovery to determine if we can remove deployments #2580 - Ponnia and Lisa 
- [x] 13. Synthesize the partner team survey #2842 @bianca-rivera 
- [x] 14. Synthesize the retro findings #2855 @bianca-rivera 
- [x] 15. Investigate GH workflow failure for helm chart domain-cc on default branch #2862- Lisa

16. ~~Make a recommendation to improve the deployment request process #2672 @bianca-rivera and Erik (won't have enough capacity to get to in Sprint S, move to future sprint~~ 

### Stretch Goals:
The following list are issues that can be worked on after the priority issues have been completed or addressed. **See Megan or Berni with questions before pulling in unplanned work.**
- Get RBAC deployed to Sandbox and Prod #2846 and #2560
- Discovery on data retention for VRO #2823 

### Unplanned Work:
These are issues that were added to the current sprint in order to unblock current issues.
- #2862 - Lisa
